pkg/cli/options_test.go

```diff
@@ -17,16 +17,279 @@ limitations under the License.
 package cli
 
 import (
+	"errors"
+	"os"
+	"path/filepath"
+	"runtime"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
 	"sigs.k8s.io/kubebuilder/v3/pkg/model/stage"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 )
 
+// func TestExternalPluginsDiscovery(t *testing.T) {
+// 	RegisterFailHandler(Fail)
+// 	RunSpecs(t, "TestExternalPluginsDiscovery")
+// }
```

Comment:
@rashmigottipati why it is commented? Are this test been called?

Reply:
Suite setup for cli is already handled [here](https://github.com/kubernetes-sigs/kubebuilder/blob/e8f2ae4537b3bb996018c1f7b3ef18db33885118/pkg/cli/suite_test.go#L29-L32) so this can be removed.

Reply:
Yes exactly, that's the reason I commented it out. Will remove it. Was using it mainly for running the tests locally. 
Also, defining this here was causing a Gingko error too (it's a known issue for gingko/gomega) - will try to find the link for it.